apps/beeai-ui/src/modules/agents/components/AgentDetailPanel.tsx

```diff
@@ -55,13 +53,15 @@ export function AgentDetailPanel() {
                 <>
                   {description && <MarkdownContent className={classes.description}>{description}</MarkdownContent>}
 
-                  <AgentMetadata agent={agent} />
+                  {metadata.author && <span>By {metadata.author.name}</span>}
```

Comment:
This is a very detail. But could be nice to have as:
```tsx
const authorName = metadata.author.name;

// ...

{authorName && <span>By {authorName}</span>}
```

Just to follow the same pattern as we have for the `documentationUrl`. It also reads nice, and makes sure "name" is defined.